docs/developing_recipes_locally.md

```diff
@@ -124,6 +124,52 @@ It is possible to run the linter locally the same way it is being run [using Git
   pylint --rcfile=linter/pylintrc_testpackage recipes/fmt/all/test_package/conanfile.py
   ```
 
+## Running the YAML Linters
+
+There's two levels of YAML validation, first is syntax and the second is schema.
+The style rules are located in [`linter/yamllint_rules.yml`](../linter/yamllint_rules.yml) and are used to ensure consistence.
+The [`config.yml](how_to_add_packages.md#configyml) is required for the build infrastructure and the
+[`conandata.yml` patch fields](conandata_yml_format.md#patches-fields) have required elements that are enforced with
+schema validation. There's are to encourage the best possible quality of recipes and make reviewing faster.
+
+### Yamllint
+
+* (Recommended) Use a dedicated Python virtualenv.
```

Comment:
what exactly does it mean? a link would be helpful (to some `virtualenv` tutorial?)